### ISSUE TITLE
Wrong documentation about importing files and bit depth?

### ISSUE THREAD

**Issue by NightProgramming** [audacity/audacity#4919]
### Bug description

The [manual](https://manual.audacityteam.org/man/importing_audio.html) says: "Generally, Audacity imports files at the sample format (bit depth) specified at Default Sample Format in Audio Settings Preferences."

But when setting the "Default Sample Format" to "24-bit" (under Edit -> Preferences -> Audio Settings) a 24 bit wav file still gets imported with a 32 bit sample rate. This contradicts the quoted statement from the docs. Also the table below the quoted statement in the docs contradicts the quoted statement.

Using Audacity 3.3.3

### Steps to reproduce

1. Read the statement "Generally, Audacity imports files at the sample format (bit depth) specified at Default Sample Format in Audio Settings Preferences." in the manual regarding "Sample Rate and Sample Format of imported files".
2. Try it out.
3. Find out, it doesn't work.

### Expected behavior

The manual and the actual behavior should not be different.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

**Comment by LWinterberg** [comment]
If you scroll down a bit more, it'll also tell you this: 

![image](https://github.com/audacity/audacity/assets/87814144/3ac4e9d8-c2da-49b3-a404-3a8fccbfc450)

**Comment by NightProgramming** [comment]
Yes, it's just about the one sentence (sorry if this was not very clear). As a newbie I read the sentence and then looked at the table, and was confused and thought the table must be wrong, and then tried it out and found out the sentence is wrong. So the overall experience was a bit confusing (and time consuming).
